pkg/cmd/service/service_test.go

```diff
@@ -612,7 +614,8 @@ func TestHTTPServingTLS(t *testing.T) {
 }
 
 func TestGRPCServingTLS(t *testing.T) {
-	logger := logger.NewNoopLogger()
+	logger, err := logger.NewTextLogger()
```

Comment:
?

Reply:
Ah, was using for debugging. Reverted. Thanks.